data/overview.yaml

```
@@ -10,6 +10,7 @@ General algebra:
     comma category: 'category_theory.comma'
     limits: 'category_theory.limits.is_limit'
     presheaved spaces: 'algebraic_geometry.PresheafedSpace'
+    sheaved spaces: 'algebraic_geometry.SheafedSpace'
```

```suggestion
    presheafed spaces: 'algebraic_geometry.PresheafedSpace'
    sheafed spaces: 'algebraic_geometry.SheafedSpace'
```
(Is this more standard?)